setup.py

```diff
@@ -0,0 +1,22 @@
+import setuptools
+
+with open("README.md", "r") as fh:
+    long_description = fh.read()
+
+setuptools.setup(
+    name="majortom_gateway",
+    version="0.0.2",
+    author="Jesse Coffey",
+    author_email="jcoffey@kubos.com",
```

Comment:
Consider making an email alias for open-source@kubos.com or something.

Reply:
Also a good idea.